cli/js/lib.deno.ns.d.ts

```diff
@@ -1036,7 +1037,7 @@ declare namespace Deno {
    *       Deno.mkdirSync("nested/directories", { recursive: true });
    *       Deno.mkdirSync("restricted_access_dir", { mode: 0o700 });
    *
-   * Throws error if the directory already exists.
+   * Defaults to throwing error if the directory already exists.
```

Comment:
Why “defaults to”?

Reply:
It was confusing that the comments on `MkdirOptions.recursive` say (correctly) that it doesn't throw when a directory already exists, but then the comments on `mkdir` imply it does throw then.

Reply:
that is, doesn't throw when `recursive` is true

Reply:
I see. It would be clearer if it said that explicitly.

"If recursive is false, mkdir will throw if the directory already exists"